src/BeanstalkClient.php

```diff
@@ -49,6 +50,18 @@ public function __construct(string $uri) {
                 }
             }
         });
+        $this->connection->addEventHandler("close", function () {
+            if ($this->isConnected) {
```

Comment:
Cloud you explain why we need this boolean?

Reply:
This boolean shows state of connection. It needs to determine was a connection closed by client or by server.

Reply:
You are right. It was unnecessary boolean.